consensus/cometbft/service/encoding/encoding.go

```diff
@@ -81,7 +83,14 @@ func UnmarshalBeaconBlockFromABCIRequest(
 		return signedBlk, ErrNilBeaconBlockInRequest
 	}
 
-	return ctypes.NewSignedBeaconBlockFromSSZ(blkBz, forkVersion)
```

Comment:
similarly to what we do with blobs, we use decoder instead of yet another constructor